wrappers/javascript/yarn.lock

```diff
@@ -1288,6 +1143,7 @@
     "@types/ref-struct-di" "*"
 
 "@types/2060.io__ref-napi@npm:@types/ref-napi", "@types/ref-napi@*":
```

Comment:
CI still failing 😭. It looks like it will also need to clean yarn lock file + `yarn install` for it to properly fetch the new types dependencies.